projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -38,26 +42,15 @@
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'SignedRelease' ">
-    <DelaySign>true</DelaySign>
-    <OutputType>Library</OutputType>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
     <Optimize>true</Optimize>
```

Comment:
You don't need to set `Optmize` at all, so you can remove everything related to that.